src/chip.tsx

```diff
@@ -39,29 +41,66 @@ const useStyles = createUseStyles(({colors, mq}) => ({
             },
         },
```

Comment:
weird colors in prominent and classic o2 while hover
![image](https://user-images.githubusercontent.com/6722153/158555513-4b270759-234f-4dc5-acb0-ff96be947cc5.png)
![image](https://user-images.githubusercontent.com/6722153/158555533-1ebf36a3-8736-4ae0-88e5-94e40acceebd.png)

I'm going to check constant usage